ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAPP-662] Add cannot check with dwp screen

## Description of change
Add new cannot check dwp status screen 
Removes and replaces previous retry benefit check screen (see screenshot below)

## Link to relevant ticket
[CRIMAPP-662](https://dsdmoj.atlassian.net/browse/CRIMAPP-662)

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:
<img width="932" alt="Screenshot 2024-04-08 at 15 19 07" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/d71193ca-7d7b-45fa-bf95-2f5004082a91">


### After changes:
<img width="932" alt="Screenshot 2024-04-08 at 16 11 22" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/b53c5dae-f265-4b29-9114-6f9ff5be567a">

## How to manually test the feature
Note you will need to set the env variable `BC_USE_DEV_MOCK` to `false` to simulate the dwp api being down 
Test the following scenario with the feature flag turned on and then turned off  

1. Start an application 
2. Enter a nino
3. Select a benefit type 
4. You should see the screen `Sorry we cannot check your client's info with DWP`
5. Click save and continue and you should be routed to the screen `Do you have evidence your client gets ____?`
6. Go back and click save and come back later and you should be routed to the task list 

[CRIMAPP-662]: https://dsdmoj.atlassian.net/browse/CRIMAPP-662?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
